index.js

```diff
@@ -109,6 +109,7 @@ function buildAssets (opts, callback) {
             }
             rawScss.push('@import "'+opts.variables+'";');
         }
+        rawScss.push('@import "'+getModulePath('slate')+'/source/stylesheets/_variables.scss";');
```

Comment:
I think this should be before the if condition -> you want to be able to overwrite the variables